src/microbe_stage/systems/EngulfedDigestionSystem.cs

```diff
@@ -240,6 +247,9 @@ protected override void Update(float delta, in Entity entity)
                 if (totalAmountLeft <= 0 || engulfable.DigestedAmount >= Constants.FULLY_DIGESTED_LIMIT)
                 {
                     engulfable.PhagocytosisStep = PhagocytosisPhase.Digested;
+
+                    if (entity.Has<PlayerMarker>() && engulfedObject.Has<MicrobeControl>())
+                        gameWorld?.StatisticsTracker.TotalEngulfedByPlayer.Increment(1);
```

Comment:
There should be a preupdate overridden method that ensures game world is not null and this uses then null suppression to be consistent with other systems.